examples/gcoap/gcoap_cli.c

```diff
@@ -54,6 +54,10 @@ static gcoap_listener_t _listener = {
     NULL
 };
 
+/* retain request path to re-request if response includes block */
```

Comment:
What happens if normal request happens during an ongoing block-wise transfer? This would get overwritten and the blockwise transfer interrupted right?

Reply:
Yes, that's correct. As a command line app, I would not expect the user to run requests simultaneously.

A better solution is possible for confirmable messages as shown in this [comment](https://github.com/RIOT-OS/RIOT/pull/9857#issuecomment-423747128) to #9857 -- if we make the whole request memo available to the response handler. In this case we can read the Uri-Path options from the original request.

Reply:
> Yes, that's correct. As a command line app, I would not expect the user to run requests simultaneously.

Ok, I agree. Can we just add a comment specifying this? Although this is a command line app, it is an examples on which someone might base there code so a comment specifying its limitation doesn't hurt IMO.

Reply:
Sounds good. I'll add this where _last_req_path is defined.